### PR TITLE
fix SHACL-C, among other typos and grammar

### PR DESCRIPTION
**Pull request by TallTed**

### CODE REVIEW

File: shacl-compact-syntax/scope.md [hunk 1]
```diff
@@ -1,53 +1,53 @@
...
 
 ## Aligning with RDF 1.2
 
 Syntactic additions will be required for SHACL-C to support the following:
 
- - [Support for datasets](https://github.com/w3c/shacl/issues/22)
- - [Support for RDF 1.2 (in particular embedded triples)](https://github.com/w3c/shacl/issues/23)
- - [Support for RDF 1.2 text direction](https://github.com/w3c/shacl/issues/24)
+ - [RDF datasets](https://github.com/w3c/shacl/issues/22)
+ - [RDF 1.2 (in particular embedded triples)](https://github.com/w3c/shacl/issues/23)
```
Comment (afs) [inline shacl-compact-syntax/scope.md:14]:
```suggestion
 - [RDF 1.2 (occurrences and triple terms](https://github.com/w3c/shacl/issues/23)
```

**Comment by afs** [comment]
Using SHACL-C is a good idea.

**Review by afs — APPROVED** [review]

File: shacl-compact-syntax/scope.md (hunk 1 above)
Comment (TallTed) [reply shacl-compact-syntax/scope.md:14]: @afs — Good catch. Needs to lose the open-paren, too. Confirm this is satisfactory?
```suggestion
 - [RDF 1.2 triple terms and occurrences](https://github.com/w3c/shacl/issues/23)
```

Comment (afs) [reply shacl-compact-syntax/scope.md:14]: LGTM

**Review by jeswr — APPROVED** [review]

**Pull request merged**